file/StreamWrapper.php

```diff
@@ -364,4 +364,61 @@ public function stream_stat()
 
         return $statistics;
     }
+    
+    /**
+     * Seeks to specific location in a stream.
+     * This method is called in response to `fseek()`.
+     * @see fseek()
+     * @param integer $offset The stream offset to seek to.
+     * @param integer $whence 
+     * Possible values:
+     * 
+     * - SEEK_SET - Set position equal to offset bytes.
+     * - SEEK_CUR - Set position to current location plus offset.
+     * - SEEK_END - Set position to end-of-file plus offset.
+     * 
+     * @return boolean Return true if the position was updated, false otherwise.
+     */
+    public function stream_seek($offset, $whence = SEEK_SET)
+    {
+        switch ($whence) {
+            case SEEK_SET:
+                if ($offset < $this->download->getSize() && $offset >= 0) {
+                     $this->pointerOffset = $offset;
+                     return true;
+                } else {
```

Comment:
There should not be `else` after `return`.
Code should have following strucutre:

```php
if (...) {
    ...
    return ;
}
...
return;
```